integration-tests/verification-flow/range-proofs.test.ts

```diff
@@ -49,6 +52,19 @@ describe('Range proofs verification', () => {
     expect(
       presentation.verifiableCredential[0].credentialSubject,
     ).toBeUndefined();
+
+    // assert that the bounds for dateOfBirth are present
```

Comment:
should we also assert that the dateOfBirth isn't explicitly there?

Reply:
I think we already do that when checking the credentialSubject is empty

```